src/app/dataset-flow/dataset-flow-details/dataset-flow-details.component.ts

```diff
@@ -79,13 +57,22 @@ export class DatasetFlowDetailsComponent extends BaseDatasetDataComponent implem
         }/${this.flowId}/${tab}`;
     }
 
-    public get datasetInfo(): DatasetInfo {
-        return this.getDatasetInfoFromUrl();
+    private startTimer(): void {
+        timer(0, 5000)
+            .pipe(
+                skip(1),
+                filter(() => Boolean(this.flowDetails.flow.status !== FlowStatus.Finished)),
```

Comment:
Maybe the timer should be stopped once reached Finished state?

Reply:
Try this suggestion

```
private startTimer(): void {
    timer(0, 5000)
        .pipe(
            skip(1),
            takeWhile(() => this.flowDetails.flow.status !== FlowStatus.Finished),
            tap(() => {
                this.refreshNow();
            }),
            takeUntilDestroyed(this.destroyRef),
        )
        .subscribe();
}
```

Reply:
Fixed